### ISSUE TITLE
Install on Scientific Linux 6.x can't start

### ISSUE THREAD

**Issue by willhertz** [inverse-inc/packetfence#1804]
## Scenario
Using RHEL derivative Scientific linux 6.x (tested with 6.7 and 6.8)
and following the instructons on download page and on documentation
I get the following 

```**# yum install --enablerepo=packetfence packetfence**
Loaded plugins: fastestmirror, security
Setting up Install Process
Loading mirror speeds from cached hostfile
 * sl: ftp.scientificlinux.org
 * sl-security: ftp.scientificlinux.org
 * sl6x: ftp.scientificlinux.org
 * sl6x-security: ftp.scientificlinux.org
http://inverse.ca/downloads/PacketFence/RHEL6.8/x86_64/repodata/repomd.xml: [Errno 14] PYCURL ERROR 22 - "The requested URL returned error: 404 Not Found"
Trying other mirror.
To address this issue please refer to the below knowledge base article

https://access.redhat.com/articles/1320623

If above article doesn't help to resolve this issue please open a ticket with Red Hat Support.

Error: Cannot retrieve repository metadata (repomd.xml) for repository: packetfence. Please verify its path and try again
```
It was working in the morning with a test install over Centos7 but not with Scientific linux 6.x

## What can be wrong?
Seems like the mirror can't resolve some sort of address.
The `etc/yum.repos.d/packectfence` 
references download path "automagically" making use of yum variable `$releasever`
like this
```
[packetfence]
name=PacketFence Repository
baseurl=http://inverse.ca/downloads/PacketFence/RHEL$releasever/$basearch
gpgcheck=0
enabled=0
```
The directories on mirror server are named only after Major release i.e. RHEL7 RHEL6, 
**Using the following in Centos7** (the python one-liner reads the yum variables and shows them on stdout)
>`# python -c 'import yum, pprint; yb = yum.YumBase(); pprint.pprint(yb.conf.yumvar, width=1)'`

shows
```
Loaded plugins: fastestmirror
{'arch': 'ia32e',
 'basearch': 'x86_64',
 'infra': 'stock',
 'releasever': '7',
 'uuid': 'some-mumbo-jumbo'}
```
Repeating same procedure in Scientific Linux 6.8

>`# python -c 'import yum, pprint; yb = yum.YumBase(); pprint.pprint(yb.conf.yumvar, width=1)'`
shows
``` 
Loaded plugins: fastestmirror
{'arch': 'ia32e',
 'basearch': 'x86_64',
 'releasever': '6.8',
 'uuid': 'more-mumbo-jumbo'}
```
**BINGO!!!**
releasever expands to 6.8 so it resolves to a non-existent directory
`http://inverse.ca/downloads/PacketFence/RHEL6.8/x86_64/`
instead  of
`http://inverse.ca/downloads/PacketFence/RHEL6/x86_64/`
that does exist.

## Possible Solutions (on developers side):

- Do some regexp in packetfence.repo to extract only the major-release portion from $releasever variable (seems risky, prone to failure, boring and I don't even know if possible)
- Make symlink or http redirect in mirror server that intercepts the dot and the minor-release from $releasever variable and redirects to the right path (Seems feasible)
## Inmediate solution on users side.
- Create the file /etc/yum/vars/releasever and put number `6` as only content on the first line. (Works ASAP, but not sure if it may interfere with yum and other repositores that expect newer systems than 6 flat. And if the file is removed if it may interfere with packetfence updating)

If someone thinks on a better solution please let know and update the docs accordingly.
Best Regards

**Comment by julsemaan** [comment]
We already support CentOS 6 and 7 as well as Debian 7 and 8

Trust me, our packaging team already has enough work to keep them busy and enough packaging headaches so even though it might be easy to support Scientific Linux, I don't think we'll want to do it as we're stuck with fixing the problems with it afterwards.

Its the same thing as Fedora where CentOS is Fedora at a certain point in time.
That doesn't mean we'll support Fedora even though it might work

@extrafu, @fdurand and @louismunro can probably provide additionnal input on this as they are closer to packaging than I am but IMO this should be a 'closed won't fix'

**Comment by dwlfrth** [comment]
@willhertz: As @julsemaan said, packaging job is almost a full-time job (kidding but still, it requires a lot of time... and patience). I still think you made a good effort on finding solutions to the "issue" and would encourage you to post your findings on the PacketFence-users mailing-list (https://lists.sourceforge.net/lists/listinfo/packetfence-users) so that other could benefit from it even if it does not make it into official packaging.
Cheers! 🍻
